client/src/app/components/client-gridview/client-gridview.component.ts

```diff
@@ -55,4 +67,36 @@ export class ClientGridviewComponent implements OnInit {
       this.changeDetectorRefs.detectChanges();
   }
 
+  // Modifies the user by calling the API and then refreshes the table
+  updateUser(id: number, property: string, event: any): void {
+    let editField;
+    // Dropdown values are sent as events, editable text such as the names are sent via string
+    if (typeof event === 'string' || typeof event === 'boolean')
+    {
+      editField = event;
+    }
+    else
+    {
+      editField = event.target.textContent;
+    }
+    this.manageMembersService.modifyClientInfo(property, editField, id).subscribe(
+    (response) => {
+      this.users[id-1][property] = response[property];
+    },
+    (err) => {
+      // If name contains illegal characters
+      if (err.error.last_name) {
+        this.errorMessage = err.error.last_name[0];
+      }
+
+      // If name contains illegal characters
+      if (err.error.first_name) {
+        this.errorMessage = err.error.first_name[0];
+      }
+    });
+  }
+
+  changeValue(id: number, property: string, event: any): void {
```

Comment:
Why do you still have the id and property arguments of the function if you are not using them.